django_extensions/management/commands/shell_plus.py

```diff
@@ -203,7 +204,7 @@ def generate_kernel_specs(self, app, ipython_arguments):
 
         return {'django_extensions': ks}
 
-    def run_notebookapp(self, app, options, use_kernel_specs=True):
+    def run_notebookapp(self, initialize, options, use_kernel_specs=True):
```

Comment:
Can be use something else then `initialize` here ? maybe `app_init` or something. Which would more clearly describe it's purpose...

Reply:
done